cla_common/services/__init__.py

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Spacing in this file is tabs. Should be spaces as everywhere else.

(suspect it may have been your text editor when pasting into a new file before saving it with a .py extension, so it wasn't using the normal configuration)

Reply:
updated, please have a look again